gdpr/gdpr/AppDelegate.m

```diff
@@ -0,0 +1,63 @@
+//
+//  AppDelegate.m
+//  gdpr
+//
+//  Created by Kaleb Bataran on 4/3/18.
+//  Copyright © 2018 localytics. All rights reserved.
+//
+
+#import "AppDelegate.h"
+@import Localytics;
+
+@interface AppDelegate ()
+
+@end
+
+@implementation AppDelegate
+
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    // Override point for customization after application launch.
+    
+    [Localytics pauseDataUploading:YES];
+    [Localytics autoIntegrate:@"a39f6f0cb378dbd8239767d-487f3e04-8691-11e6-ab4e-0098ae15fcaa"
+        withLocalyticsOptions:@{
+                                LOCALYTICS_WIFI_UPLOAD_INTERVAL_SECONDS: @5,
+                                LOCALYTICS_GREAT_NETWORK_UPLOAD_INTERVAL_SECONDS: @10,
+                                LOCALYTICS_DECENT_NETWORK_UPLOAD_INTERVAL_SECONDS: @30,
+                                LOCALYTICS_BAD_NETWORK_UPLOAD_INTERVAL_SECONDS: @90
+                                }
+                launchOptions:launchOptions];
```

Comment:
You need to check if the user is logged in here.  If they are logged in then you should be checking their opt in status on a server.